feat: add employees component

punwai opened the pull request:
## Description
- Add employees component

## Related issues and discussion
#156

## Screenshots, if any
![employee](https://user-images.githubusercontent.com/28529491/70376579-09025c00-1902-11ea-876e-8cc3b2c1f2d3.gif)

## Checklist
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] If you have multiple commits please combine them into one commit by squashing them.

- [x] Read and understood the contribution guidelines at `web-app/.github/CONTRIBUTING.md`.

abhaychawla commented on src/app/organization/employees/employees.component.html:
```diff
@@ -0,0 +1,61 @@
+        <th mat-header-cell *matHeaderCellDef mat-sort-header> Status </th>
+        <td mat-cell *matCellDef="let employees">
+          <div [className]="employees.isActive === true ? 'active' : 'inactive'">
+            <fa-icon matTooltip="{{ employees.isActive === true ? 'Active' : 'Inactive' }}" matTooltipPosition="right" icon="stop" size="lg"></fa-icon>
```
Let's update this (icon, colors whatever necessary) as per the components currently implemented in the web app: for example, System => Hooks, etc. to ensure consistency. We are using circular icons here.

abhaychawla commented on src/app/organization/employees/employees.component.scss:
```diff
@@ -0,0 +1,15 @@
+}
+
+.inactive {
+  color: #666f73;
```
```suggestion
  color: #f44366;
```

abhaychawla commented on src/app/organization/employees/employees.component.html:
```diff
@@ -0,0 +1,61 @@
+      </ng-container>
+
+      <ng-container matColumnDef="isLoanOfficer">
+        <th mat-header-cell *matHeaderCellDef mat-sort-header> Is Loan Officer </th>
```
```suggestion
        <th mat-header-cell *matHeaderCellDef mat-sort-header> Loan Officer </th>
```

abhaychawla commented on src/app/organization/employees/employees.component.html:
```diff
@@ -0,0 +1,61 @@
+
+      <ng-container matColumnDef="isLoanOfficer">
+        <th mat-header-cell *matHeaderCellDef mat-sort-header> Is Loan Officer </th>
+        <td mat-cell *matCellDef="let employee"> {{ employee.isLoanOfficer }} </td>
```
Let's update this with icons similar to what's currently implemented in the web app components like System => Codes etc.

abhaychawla replied:
@punwai For loan officer as I specified take System => Codes as an example where we have icons `check-circle` and `times-circle`.

punwai commented:
I have added the necessary changes. I have:
- Changed the icons to circular icons
- Changed Loan Officer to circular icons
- Loan Officer Tooltip: Yes, No
- Status Tooltip: Active, Inactive

<img width="1001" alt="Screenshot 2019-12-07 at 23 12 54" src="https://user-images.githubusercontent.com/28529491/70381713-4c32ee00-1947-11ea-87b0-e69dac2e02e5.png">
<img width="153" alt="Screenshot 2019-12-07 at 23 13 08" src="https://user-images.githubusercontent.com/28529491/70381714-4ccb8480-1947-11ea-8d59-eeb518cd4c40.png">
<img width="160" alt="Screenshot 2019-12-07 at 23 13 11" src="https://user-images.githubusercontent.com/28529491/70381715-4ccb8480-1947-11ea-8d9e-651e3e788440.png">

punwai commented:
<img width="186" alt="Screenshot 2019-12-08 at 09 18 49" src="https://user-images.githubusercontent.com/28529491/70387332-0ce2bc80-199c-11ea-9ae1-8c3a4b34208c.png">

punwai commented:
Made the necessary changes :)
